src/User/User.php

```diff
@@ -876,4 +877,26 @@ public function refreshDiscussionCount()
 
         return $this;
     }
+
+    /**
+     * Set a user as the actor on a request instance.
+     *
+     * @param Request $request
+     * @return
+     */
+    public static function setRequestActor(self $actor, Request $request): Request
```

Comment:
nitpick: dockblock isn't accurate

Reply:
I'll fix it when we settle on naming / structure; any thoughts on that?

Reply:
Oh I didn't see that, my bad